eva/udfs/yolo_object_detector.py

```diff
@@ -41,10 +44,14 @@ class YoloV5(PytorchAbstractClassifierUDF):
     def name(self) -> str:
         return "yolo"
 
-    def setup(self, threshold=0.85):
+    def setup(self, threshold=0.60):
         logging.getLogger("yolov5").setLevel(logging.CRITICAL)  # yolov5
         self.threshold = threshold
-        self.model = torch.hub.load("ultralytics/yolov5", "yolov5s", verbose=False)
+        self.predict_func = YOLO("yolov8s.pt")
```

Comment:
use more accurate model

Reply:
Okay, should I use the Medium one or large model?